src/controllers/api/v1/tickets.js

```diff
@@ -529,17 +529,18 @@ apiTickets.create = function (req, res) {
  }
  */
 apiTickets.createPublicTicket = function (req, res) {
-  var Chance = require('chance')
+  const Chance = require('chance')
 
-  var chance = new Chance()
-  var response = {}
+  const chance = new Chance()
+  const response = {}
   response.success = true
-  var postData = req.body
-  if (!_.isObject(postData)) return res.status(400).json({ success: false, error: 'Invalid Post Data' })
-
-  var user, group, ticket, plainTextPass
+  const postData = req.body
+  if (!_.isObject(postData)) {
+    return res.status(400).json({ success: false, error: 'Invalid Post Data' })
+  }
+  let user, group, ticket, plainTextPass
 
-  var settingSchema = require('../../../models/setting')
+  const settingSchema = require('../../../models/setting')
```

Comment:
is it possible to use absolute path? If that configuration is not done, we can leave it.